extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -381,10 +383,99 @@ class MembershipsButtonEdit extends Component {
 		);
 	};
 
-	render = () => {
+	renderUpgradeNudges = () => {
```

Comment:
This is called `renderUpgradeNudges`, but includes more than pre-purchase/connection nudges now; maybe we could split this into two?

Reply:
Done :+1: